src/app/rundown-view/factories/key-binding.factory.ts

```diff
@@ -38,32 +41,61 @@ export class KeyBindingFactory {
   }
 
   private createInsertCameraOnAirKeyBinding(cameraAction: Tv2CameraAction, rundownId: string): StyledKeyBinding {
-    const cameraNumber: number = cameraAction.metadata.cameraNumber
+    const cameraNumber: number = parseInt(cameraAction.metadata.cameraNumber)
+    if (Number.isNaN(cameraNumber)) {
+      throw new Error('Expected camera number to be an integer.')
+    }
     return {
-      keys: [`Digit${cameraNumber}`],
+      keys: [`F${cameraNumber}`],
```

Comment:
I assume `F` is the function key prefix?
Could be nice to have that extracted to a const.

Reply:
Correct. The `F` is extracted to the constant `FUNCTION_KEY_PREFIX`.